README.md

```diff
@@ -462,6 +466,30 @@ When you require `react-native-code-push`, the module object provides the follow
 
 * [sync](#codepushsync): Allows checking for an update, downloading it and installing it, all with a single call. Unless you need custom UI and/or behavior, we recommend most developers to use this method when integrating CodePush into their apps
 
+#### codePush.allowRestart
+
+```javascript
+codePush.allowRestart(): void;
+```
+
+Re-allows programmatic restarts to occur, that would have otherwise been rejected due to a previous call to `disallowRestart`. If `disallowRestart` was never called in the first place, then calling this method will simply result in a no-op.
```

Comment:
As you mentioned, this is ordered alphabetically, not based on the sequence you would call them in. I don't see the reason why we would order things based on sequence in an API reference document. Additionally, as I call out, this is already an advanced API, so I don't think we need to optimize our doc flow specifically for this.